modules/ROOT/pages/mule-app-properties-to-configure.adoc

```diff
@@ -81,6 +81,73 @@ image::mruntime-properties-file.png[properties-file]
 </http:request-config>
 ----
 
+When the application is deployed, the property values will be computed from the different possible sources and will statically replace the placeholders as if they have been written directly in the source code.
+
+Following the previous example, it would be equivalent to have written it like this:
+
+[source, xml]
+----
+<http:request-config
+  name="HTTP_Request_config"
+  doc:name="HTTP Request
+  configuration" doc:id="7120494c-0540-4ad1-a118-f5b6db3f1456"
+  basePath="/service" >
+		<http:request-connection
+      host="my-api.cloudhub.io"
+      port="10000" />
+</http:request-config>
+----
+
+There is no special escaping based on the context, so the resulting values need to be valid in the context from which they are referenced.
+
+Consider, for example, the following scenario which contains a Dataweave script.
+
+[source, xml]
+----
+<ee:transform>
+  <ee:message>
+    <ee:set-payload>
+      <![CDATA[
+        %dw 2.0
+        output application/json
+        ---
+        {
+          "message": "${message}",
+        }
+      ]]>
+    </ee:set-payload>
+  </ee:message>
+</ee:transform>
+----
+
+And the contents of the `.properties` file look like this:
+
+----
+message=I love $
+----
+
+After property replacements take place, the application would look like this:
+
+[source, xml]
+----
+<ee:transform>
+  <ee:message>
+    <ee:set-payload>
+      <![CDATA[
+        %dw 2.0
+        output application/json
+        ---
+        {
+          "message": "I love $",
+        }
+      ]]>
+    </ee:set-payload>
+  </ee:message>
+</ee:transform>
+----
+
+This is an invalid Dataweave script because the dollar sign has a special meaning and needs to be escaped. See xref:dataweave::dataweave-types.adoc#escape-chars-string[Dataweave - Escaping Special Characters].
```

Comment:
```suggestion
The output is an invalid DataWeave script because the dollar sign ($) has a special meaning and needs to be escaped. See xref:dataweave::dataweave-types.adoc#escape-chars-string[Dataweave - Escaping Special Characters].
```